lib/ws-channel.js

```diff
@@ -289,11 +294,13 @@ WebsocketChannel.prototype._closeWebsocket = function(websocket) {
 
 
 WebsocketChannel.prototype.notify = function(message) {
-  this._debug('notify: %s', this._debug.json(message));
+  var requestId = this._requestId++;
+
+  this._debug('queue not %d: %s', requestId, this._debug.json(message));
```

Comment:
After a minute I decided that `not` was short for `notification`.. reading this expression split across two lines (assuming 'notification' is long enough to cause a wrap) would be much better use of code reader's time ;-)